# Motor driver configuration for the BOOM

motor0:
  id: 0
  control_mode: 3
  input_mode: 1
  gear_ratio: 6.0
  can_id: 0
  pos_gain: 25.0
  vel_gain: 0.15
  acc_gain: 0.30
  velocity_limit: 20.0
  current_limit: 5.0

motor1:
  id: 1
  control_mode: 3
  input_mode: 1
  gear_ratio: 6.0
  can_id: 1
  pos_gain: 25.0
  vel_gain: 0.15
  acc_gain: 0.30
  velocity_limit: 20.0
  current_limit: 5.0

# Control Modes
# 1: Torque control
# 2: Velocity control
# 3: Position control

# Input Modes
# 1: Passthrough (Any control mode)
# 2: Velocity Ramp (Control mode 2 only)
# 3: Position filter (Control mode 3 only)
# 5: Trapezoidal Trajectory (Control mode 3 only)
# 6: Torque Ramp (Control mode 1 only)